.circleci/config.yml

```diff
@@ -59,9 +59,10 @@ jobs:
           name: Run Tests
           command: |
             . ~/venv/bin/activate
+            pip install gitpython==3.1.7
```

Comment:
could you add that to requirements.txt instead?

Reply:
I've tried that and it didn't work: https://app.circleci.com/pipelines/github/joeyzhou98/conp-dataset/12/workflows/a2c334f2-445b-4157-8e57-b6f515130341/jobs/33

Reply:
ok then...